jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -223,11 +224,19 @@ def installationManifestFile = './evals/inventories/group_vars/all/manifest.yaml
 currentBuild.displayName = "${currentBuild.displayName} ${productName}"
 
 def ensureEvalsDir() {
-    if(!fileExists("evals")) {
+    if (!fileExists("evals")) {
         sh "ln -s . evals"
     }
 }
 
+def updateManifestVariable(manifestFileTxt, name, value) {
+    if (name && value) {
+        manifestFileTxt.replaceFirst(/${name}: '.*'/, "${name}: '${value}'")
+    } else {
+        println("Unable to update manifest variable: name = ${name}, value = ${value}")
+    }
+}
```

Comment:
Added generic method to update a key/value pair in the manifest file if both exist.